content/blog/2019-08-15-new-react-devtools.md

```diff
@@ -4,7 +4,7 @@ author: [bvaughn]
 ---
 We are excited to announce a new release of the React Developer Tools, available today in Chrome, Firefox, and (Chromium) Edge!
 
-## What's changed?
+## What's changed? {#whats-changed}
```

Comment:
`## What's changed? {#whats-changed}` => `## What's changed?`

since you are not working on this page. let's revert this change